build-patch-filter.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+git diff HEAD~1 HEAD > show.patch
```

Comment:
`git diff HEAD~1 HEAD > target/show.patch`

to not create any extra files in git repo and keep `git diff` empty.

Reply:
done